### ISSUE TITLE
Feature: Use graphical key / button representations in the HUD

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/community-rpg#7]
## tl;dr:
The HUD would be easier to understand if we used icons for keys/buttons instead of text.

### What we have now
Per [discord discussion](https://discord.com/channels/458662222697070613/704736572603629589/968276276681007154) with @Cleptomania:
- the current HUD has slots numbered 1 through 10 at the bottom of the screen
- the slots were originally intended to correspond to the `1` through `0` keys
- item icons are supposed to go in those slots

![image](https://user-images.githubusercontent.com/36696816/165186577-46dc1feb-bf3d-4429-934c-e0dbb63c04b0.png)

### What would be better
1. Use graphical assets that look like keys and buttons instead of text labels
2. Optional: integrate it with #3 

### Implementation Suggestions
For the icons, [Kenney.nl has a nice example we could scale up for use in this project](https://kenney.nl/assets/input-prompts-pixel-16):

![image](https://user-images.githubusercontent.com/36696816/165187051-9280f77b-4701-4d42-bc32-a8658ecc8936.png)

It's provided under a [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/), so the license terms are fine. Although we don't have to, we should probably still cite the asset in the README.md or other appropriate location.

**Comment by martin-martin** [comment]
I'll give this one a try.
